storage/utils/inputSerializers.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+/* eslint-disable no-unused-vars */
+
+module.exports = {
+	default: (value, mode, alias, fieldName) => `$\{${alias}}`,
```

Comment:
no where else is using `default` export, prefer to use `default` consistently

Reply:
@shuse2 `default` here is just the name (label) of the serializer type.

Reply:
yea i know, but essentially if we use `babel` style, this is how they will produce it.
maybe avoid `default`?

Reply:
Done